lib/ansible/modules/network/ovs/openvswitch_db.py

```diff
@@ -125,8 +144,15 @@ def map_config_to_obj(module):
     match = re.search(r'^' + module.params['col'] + r'(\s+):(\s+)(.*)$', out, re.M)
 
     col_value = match.group(3)
+
+    # Map types require key argument
+    has_key = module.params['key'] is not None
+    is_map = col_value[0] == '{'
+    if is_map and not has_key:
+        module.fail_json(msg="missing required arguments: key")
```

Comment:
Probably this error message can be more informative by saying missing required argument : key for map type of column